packages/arduino_ide.rb

```diff
@@ -0,0 +1,66 @@
+require 'package'
+require 'redef'
+
+redef("CREW_NOT_STRIP", "true")
```

Comment:
If you redefine, shouldn't you also undefine after usage, otherwise, doesn't it stay redefined?  In other words, the next package you attempt to install will not strip.

Reply:
No. The next package still strips.